core/storage/base_model/gae_models.py

```diff
@@ -1214,6 +1221,15 @@ class BaseSnapshotContentModel(BaseModel):
     # The snapshot content, as a JSON blob.
     content = ndb.JsonProperty(indexed=False)
 
+    @staticmethod
+    def get_deletion_policy():
+        """The content models do not contain any user ID fields directly,
+        the user ID fields might be hidden inside the content field (because
```

Comment:
Give more explanation here about what an example of such a user ID field is, and point to where "this is handled in the wipeout service". (Also, be more specific about what exactly is handled in the wipeout service?)